main/getting-started/ertp-introduction.md

```diff
@@ -12,83 +12,94 @@ object. If it doesn't have a reference, it can't.
 
 ## Creating assets with ERTP
 
-In ERTP, *mints* create digital *assets*. Access to an asset type's
-mint lets you create more digital assets of that type. You can then
+In ERTP, *mints* create *digital assets*. Access to an asset kind's
+`mint` lets you create more digital assets of that kind. You can then
 store new assets in a *payment* or a *purse*. 
-- **Payments**: Assets you intend to send to someone else.
-- **Purses**: Store assets until you withdraw them into a payment for use
+- `Payments`: Assets you intend to send to someone else.
+- `Purses`: Store assets until you withdraw them into a payment for use
 
 To send assets in ERTP:
-1. Withdraw them from a purse. This creates a payment.
-2. Send this payment to a recipient object as a message.
+1. Withdraw them from a `purse`. This creates a `payment`.
+2. Send this `payment` to a recipient object as a message.
 
 To receive assets in ERTP:
-1. Create a purse for the asset type you'll receive. **Note:** You
-do not need access to the type's mint. The Issuer (which is widely shared) provides an empty purse for the asking.
-2. Get access to the asset type you'll receive.
-3. Receive the message with the payment and deposit the payment in
-your purse.
+1. Create a `purse` for the asset kind you'll receive. Since
+an empty `purse` conveys no asset rights, you create it with
+a less powerful `Issuer` instead of a `Mint`. 
+2. Get access to the asset kind you'll receive.
```

Comment:
Step 2 should removed. It doesn't make sense. If you have access to the Issuer and have created an empty purse, there's nothing else to do but receive the payment
```suggestion
```